src/FSharpPlus/Operators.fs

```diff
@@ -94,9 +94,10 @@ module Operators =
     let inline unzip (source: '``Functor<'T1 * 'T2>``) = Unzip.Invoke source : '``Functor<'T1>`` * '``Functor<'T2>``
 
     // Zips (tuple) two functors.
+    // For collections, if one collection is shorter, excess elements are discarded from the right end of the longer collection. 
     let inline zip (source1: '``ZipFunctor<'T1>``) (source2: '``ZipFunctor<'T2>``) : '``ZipFunctor<'T1 * 'T2>`` = Zip.Invoke source1 source2
```

Comment:
@rafal I can see this is where your doc probs initiated.

These comments should be `///` not `//. If you don't mind, could you fix these also?

Reply:
No problem :-)